keywords.txt

```diff
@@ -7,16 +7,15 @@
 #######################################
 
 Node	KEYWORD1
-CallbackFunc	KEYWORD1
+Publisher KEYWORD1
```

Comment:
@OpusK you used the incorrect field separator on this line as well as 11 and 13. You must use a single true tab, not spaces.

Reference:
https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification#keywords